### ISSUE TITLE
[BUG] Default EditGrid layout does not display components in columns

### ISSUE THREAD

**Issue by hisuwh** [formio/formio.js#3128]
## Environment
Please provide as many details as you can:

- Hosting type
  - [x] Form.io
  - [x] Local deployment
    - Version: vue-formio@4.0.4
- Formio.js version: 4.11.0
- Frontend framework: Vue
- Browser: Chrome 
- Browser version: 84.0.4147.105

## Steps to Reproduce
1. Add an edit grid to a form
2. Add columns to the edit grid
3. Add components to the columns

## Expected behavior

Default layout should display components from columns

## Observed behavior

It shows the columns component and blank value for the row

## Example

Sandbox Id: 5f313364b460ba007fd216a9

Json:
```json
{
  "display": "form",
  "components": [
    {
      "label": "Edit Grid",
      "tableView": false,
      "rowDrafts": false,
      "key": "editGrid",
      "type": "editgrid",
      "input": true,
      "components": [
        {
          "label": "Columns",
          "columns": [
            {
              "components": [
                {
                  "label": "Name",
                  "tableView": true,
                  "key": "name",
                  "type": "textfield",
                  "input": true,
                  "hideOnChildrenHidden": false
                }
              ],
              "width": 6,
              "offset": 0,
              "push": 0,
              "pull": 0,
              "size": "md"
            },
            {
              "components": [
                {
                  "label": "Age",
                  "mask": false,
                  "spellcheck": true,
                  "tableView": false,
                  "delimiter": false,
                  "requireDecimal": false,
                  "inputFormat": "plain",
                  "key": "age",
                  "type": "number",
                  "input": true,
                  "hideOnChildrenHidden": false
                }
              ],
              "width": 6,
              "offset": 0,
              "push": 0,
              "pull": 0,
              "size": "md"
            }
          ],
          "tableView": false,
          "key": "columns",
          "type": "columns",
          "input": false
        }
      ]
    },
    {
      "type": "button",
      "label": "Submit",
      "key": "submit",
      "disableOnInvalid": true,
      "input": true,
      "tableView": false
    }
  ]
}
```
![image](https://user-images.githubusercontent.com/4470351/89779557-8fadaf80-db07-11ea-929e-72c484cb95c0.png)

**Comment by hisuwh** [comment]
I can fix the headers by using the `utils.eachComponent` function:

```diff
<div class="row">
-  {% (component.components || []).forEach(function(component) { %}
+  {% utils.eachComponent((component.components || []), function(component) { %}
    <div class="col-sm-2">{{ component.label }}</div>
  {% }) %}
</div>
```

![image](https://user-images.githubusercontent.com/4470351/89779971-6b9e9e00-db08-11ea-9eaf-19f61855db2b.png)

But I cannot work out how to fix the RowTemplate.  The help text says:

> Three available variables. "row" is an object of one row's data, "components" is the array of components in the grid and "state" is current row's state (can be "draft" or "saved"). To add click events, add the classes "editRow" and "removeRow" to elements.

But then the example is: 
```html
<div class="row">
  {% instance.eachComponent(function(component) { %}
    <div class="col-sm-2">
      {{ component.getView(component.dataValue) }}
    </div>
  {% }, rowIndex) %}
  {% if (!instance.options.readOnly && !instance.originalComponent.disabled) { %}
    <div class="col-sm-2">
      <div class="btn-group pull-right">
        <button class="btn btn-default btn-light btn-sm editRow"><i class="{{ iconClass('edit') }}"></i></button>
        {% if (!instance.hasRemoveButtons || instance.hasRemoveButtons()) { %}
          <button class="btn btn-danger btn-sm removeRow"><i class="{{ iconClass('trash') }}"></i></button>
        {% } %}
      </div>
    </div>
  {% } %}
</div>
```

which references `instance` which the help text doesn't identify as an available variable so I'm unsure what the type of this is.

**Comment by wag110894** [comment]
@hisuwh, 
Thank you for reaching out! We are aware of this issue and are currently working on getting this into a build. We will let you know once this issue has been resolved.

For reference, the ticket number is FJS-1130.

**Comment by abhishek0196** [comment]
Any updates on this ?

**Comment by wag110894** [comment]
This issue is resolved as a part of our 4.12.0 release.
